### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by SLS-BLN**
Just a few CSS corrections.

### CODE REVIEW

File: src/components/Blog/Blog.js [hunk 1]
```diff
@@ -100,15 +100,15 @@ const Container = styled.article`
...
-  background-color: var(--color-primary-light);
-  font-size: 1.2rem;
+  background-color: var(--color-primary);
+  font-size: 1.4rem;
   font-weight: 500;
   border: none;
   padding: 0.25rem 0.5rem;
   border-radius: var(--radius);
   margin-bottom: 1rem;
   &:hover {
```
Comment (ahohnsen) [inline src/components/Blog/Blog.js:110]: Please keep in mind that a hover effect will not work on a mobile device 🙈

Comment (ahohnsen) [inline src/components/Blog/Blog.js:110]: As you are using a `BackButton` several times I would suggest to extract this style. It seems a bit repetitive right now 🙂

File: src/components/Blog/BlogBox.js [hunk 2]
```diff
@@ -83,15 +83,15 @@ const Paragraph = styled.p`
...
   border: none;
   padding: 0.25rem 0.5rem;
   margin-bottom: 0.01rem;
   border-radius: var(--radius);
   &:hover {
-    background-color: var(--color-primary);
+    background-color: var(--color-primary-dark);
     cursor: pointer;
   }
 `;
```
Comment (ahohnsen) [inline src/components/Blog/BlogBox.js:97]: Same as above with the `BackButton`

File: src/components/Blogtag/BlogTag.js [hunk 3]
```diff
@@ -23,7 +23,7 @@ export default function BlogTag({tagName}) {
 }
```
Comment (ahohnsen) [inline src/components/Blogtag/BlogTag.js:23]: Line 10 & 21: You should not need that fragment as you are only returning one child.

File: src/components/Blogtag/BlogTag.js [hunk 4]
```diff
@@ -23,7 +23,7 @@ export default function BlogTag({tagName}) {
...
 
 const Button = styled.button`
```
Comment (ahohnsen) [inline src/components/Blogtag/BlogTag.js:25]: Same here :) extract this style and import the Button where ever you need it.

File: src/components/Blogtag/BlogTag.js [hunk 5]
```diff
@@ -23,7 +23,7 @@ export default function BlogTag({tagName}) {
...
-  font-size: 1.2rem;
+  font-size: 1.4rem;
   font-weight: 500;
   color: var(--color-font);
   border-radius: var(--radius);
```
Comment (ahohnsen) [inline src/components/Blogtag/BlogTag.js:29]: Line 43 & 45: Same as above, the fragment is not needed here.

**Review by ahohnsen — APPROVED** [review]
Hey Stefan,

Nicely done 🚀 Please have a look at my comments. I strongly suggest extracting your button styles, so that your styles are reusable. Your code should be a bit shorter afterwards 🙂

File: src/components/Blog/Blog.js (hunk 1 above)
Comment (SLS-BLN) [reply src/components/Blog/Blog.js:110]: Oh yeah, that's on my list for the next few days after the graduation event.

File: src/components/Blog/Blog.js [hunk 6]
```diff
@@ -16,7 +15,7 @@ export default function Blog() {
         <Container key={blog.id}>
           <div>
             <img src={blog.imgUrl} alt={blog.imgAlt} width={355} height={200} />
-            <BackButton onClick={() => navigate(-1)}>Zurück</BackButton>
+            <BackButton />
```
Comment (JessicaLoers) [inline src/components/Blog/Blog.js:18]: 👍

File: src/components/Blog/Blog.js [hunk 7]
```diff
@@ -38,7 +37,7 @@ export default function Blog() {
               )
             )}
           </div>
-          <BackButton onClick={() => navigate(-1)}>Zurück</BackButton>
+          <BackButton />
```
Comment (JessicaLoers) [inline src/components/Blog/Blog.js:40]: 👍

File: src/components/Blogtag/BlogTag.js [hunk 8]
```diff
@@ -7,18 +7,16 @@ export default function BlogTag({tagName}) {
...
-        aria-label={`Zeige alle Artikel und Videos zum Thema ${tagName}`}
-        onClick={() => {
-          navigate(`/search/${tagName}`.toLowerCase());
-        }}
-      >
-        {tagName}
-      </Button>
-    </>
+    <Button
+      style={{backgroundColor: colorVar}}
```
Comment (JessicaLoers) [inline src/components/Blogtag/BlogTag.js:11]: This is a comment for the future Stefan 🙂
If you want to refactor your code in the future, I would suggest avoiding inline styles and using styled components with properties instead.

File: src/components/Blogtag/BlogTag.js [hunk 9]
```diff
@@ -38,12 +36,7 @@ const Button = styled.button`
 
 export function BlogTagLine({tagName}) {
   const colorVar = tagColors.find(col => col.tag === tagName).color;
-
-  return (
-    <>
-      <Paragraph style={{backgroundColor: colorVar ? colorVar : 'black'}}>{tagName}</Paragraph>
-    </>
-  );
+  return <Paragraph style={{backgroundColor: colorVar ? colorVar : 'black'}}>{tagName}</Paragraph>;
```
Comment (JessicaLoers) [inline src/components/Blogtag/BlogTag.js:39]: Same as above regarding the styled components ✌️

**Review by JessicaLoers — APPROVED** [review]
**Hi @SLS-BLN 👋**

Congrats - really nice app.
I guess this is one of, or maybe your last PR before the Abschluss Event. 🎉

I noticed that you have implemented Andrea's suggestions about the redundant Button Styling. I also noticed that you could write a few lines of code more effectively - but that might be a task for after the bootcamp. 💻

**Pull request merged**